builtin/game/item.lua

```diff
@@ -644,7 +638,6 @@ core.noneitemdef_default = {  -- This is used for the hand and unknown items
 
 	-- Interaction callbacks
 	on_place = redef_wrapper(core, 'item_place'),
-	on_secondary_use = redef_wrapper(core, 'item_secondary_use'),
```

Comment:
Don't remove it but replace it with =nil like on_use is. This might be a tiny little bit slower, but is far more explaining.